index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header-container">
+        <h1 class="header-text">Welcome</h1>
+        <a href="#"><img src="./img/menu_icon.png" alt=""></a>
+    </header>
+    <main>
+        <section class="intro-container">
+            <h2 class="intro-title">Hey there. I'm Juan Carlos</h2>
+            <h3 class="intro-subtitle">I'm a software developer</h3>
+            <p class="intro-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="links-container">
+                <a href="#"><img src="./img/twitter_icon.svg" alt="twitter"></a>
+                <a href="#"><img src="./img/linkedin_icon.svg" alt="linkedin"></a>
+                <a href="#"><img src="./img/medium_icon.svg" alt="medium"></a>
+                <a href="#"><img src="./img/github_icon.svg" alt="github"></a>
+                <a href="#"><img src="./img/angellist_icon.svg" alt="angellist"></a>
+            </div>
```

Comment:
- [ ] Excellent job adding the icons for your social media 💯 , The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly, use `<ul>` and `<li>` tags for each `<a>` tag. 😄